listenbrainz/db/feedback.py

```diff
@@ -295,3 +298,25 @@ def get_feedback_for_multiple_recordings_for_user(db_conn, user_id: int, user_na
     query = query_base + query_remaining
     result = db_conn.execute(text(query), params)
     return [Feedback(user_id=user_id, user_name=user_name, **row) for row in result.mappings()]
+
+
+def bulk_insert_loved_tracks(db_conn, user_id: int, feedback: list[tuple[int, str]], column: str = 'recording_mbid'):
```

Comment:
Instead of duplicating the queries, let's try to refactor the existing Lfm ones and reuse them for both.